tests/foreman/ui/test_activationkey.py

```diff
@@ -926,7 +926,7 @@ def test_positive_delete_with_system(session, rhel6_contenthost, default_sat):
 
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
-def test_negative_usage_limit(session, module_org, rhel6_contenthost, default_sat):
```

Comment:
@ColeHiggins2 Will this test work without rhel6_contenthost?

Reply:
@swadeley yes. this version of the test is using a context manager to get its content hosts